### DISCUSSION TITLE
How to train CRNN Vgg16 Model with my own Image words for detection and Recognition

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1191] [Q&A]
Hey!
I need some help in Training the model CRNN with custom Images it has some letter which i need to train it for my project. 
The Text Looks Like this 
![Image 44](https://user-images.githubusercontent.com/122434059/235641149-aa9120aa-946b-46f0-8073-07ca29f04054.png) 
Can anyone explain How can i train it please @felixT2K ?

**Comment by felixT2K** [comment]
Hi @Shivashankarar 👋🏼 ,

Have you already checked [HOWTO](https://github.com/mindee/doctr/tree/main/references/recognition) ? :)
Please note that we don't support text line images currently (spaces are not detected) for example if you would train on images like this the best expected output from the recognition model would be: "123H24AJAN.23JUN.23" (without spaces)

**Reply by Shiva-Ai4m** [reply]
This is  what i excepted : 12g H24A JAN.23JUN.23
Yes I have seen it but i was not able to understand clearly LIke I have to put cropped out letter images for the words used in this 
?
